applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -133,8 +153,10 @@
               ,prev = <<"4">> :: kz_term:ne_binary()
               ,next = <<"6">> :: kz_term:ne_binary()
               ,delete = <<"7">> :: kz_term:ne_binary()
+              ,rewind = <<"5">> :: kz_term:ne_binary()
+              ,fastforward = <<"8">> :: kz_term:ne_binary()
```

Comment:
How about to use this menu
```erl
              ,keep = <<"1">> :: kz_term:ne_binary()
              ,replay = <<"2">> :: kz_term:ne_binary()
              ,forward = <<"3">> :: kz_term:ne_binary()
              ,prev = <<"4">> :: kz_term:ne_binary()
              ,callback = <<"5">> :: kz_term:ne_binary()
              ,next = <<"6">> :: kz_term:ne_binary()
              ,rewind = <<"7">> :: kz_term:ne_binary()
              ,delete = <<"8">> :: kz_term:ne_binary()
              ,fastforward = <<"9">> :: kz_term:ne_binary()
```
Callback feature on PR https://github.com/2600hz/kazoo/pull/5267

Reply:
I think the biggest problem with that is that it would change the default behavior for every deployed system and user (because you changed delete).

Reply:
I think the point is that we should not change any existing keys, and since key mapping is available per mailbox, any operator could apply the mapping you suggest at any time.

Reply:
@jamesaimonetti or @lazedo can you please comment?